examples/sim_plugins.ign

```diff
@@ -99,7 +99,7 @@
 
     <plugin entity_name="vehicle_blue"
             entity_type="model"
-            filename="ignition-gazebo1-diff-drive-system"
+            filename="gz-sim1-diff-drive-system"
```

Comment:
Oh these examples have the `gz-sim` major version hardcoded, so they won't work with the latest versions. We could use this opportunity to fix it, and we should actually fix it on Citadel and Fortress too.

```suggestion
            filename="gz-sim-diff-drive-system"
```

Reply:
gz-sim1 -> gz-sim!

We might want to migrate the `.ign` extensions to `.gz` in a separate PR eventually